src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,304 @@
+import React, {PropTypes} from 'react';
+
+import {getMuiTheme} from '../styles';
+import Paper from '../paper';
+
+const Stepper = React.createClass({
+  propTypes: {
+    /**
+     * The current active step index which passed by parent component.
+     */
+    activeStepIndex: PropTypes.number,
+    /**
+     * Children should be Step type.
+     */
+    children: PropTypes.node,
+    /**
+     * The icon will show when the step is completed.
+     */
+    completedIcon: PropTypes.node,
+    /*
+     * Override inline-style of the content container.
+     */
+    containerStyle: PropTypes.object,
+    /**
+     * Create suitable icon for step base on state of the step.
+     */
+    createIcon: PropTypes.func,
```

Comment:
It isn't clear from the description what this prop does, or what the function should return. 

Also: https://github.com/callemall/material-ui/wiki/Documentation#callback-signature